include/ur_api.h

```diff
@@ -560,6 +560,21 @@ typedef struct ur_rect_region_t {
 
 } ur_rect_region_t;
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Defines known backends.
+typedef enum ur_backend_t {
```

Comment:
I wonder if it might smooth the transition to have aliases to the old names for the two enums?

I assume the numerical values are the same so its just a source change.